### DISCUSSION TITLE
The difference in Node.js versions between local development and Docker.

### DISCUSSION THREAD

**Discussion by yuki1sntSnow** [fastapi/full-stack-fastapi-template#1189] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched in the documentation/README.
- [X] I already searched in Google "How to do X" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs/README and didn't find an answer.

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```Text
In Dockerfile

# Stage 0, "build-stage", based on Node.js, to build and compile the frontend
FROM node:20 as build-stage
```

In .nvmrc
```
18
```
```


### Description

I've noticed that the Node.js version specified in the .nvmrc file is 18, while the Dockerfile uses Node 20. I think this discrepancy may lead to inconsistencies between development and production environments.

### Operating System

Linux

### Operating System Details

wsl 20.04

### Python Version

3.10.12

### Additional Context

_No response_

**Comment by alejsdev** [comment, marked as answer]
Hi @yuki1sntSnow, you're right, the node version in the `.nvmrc` file will be updated to 20 which is the last with long-term support. Thank you for the report. :rocket:

**Reply by alejsdev** [reply]
PR: https://github.com/tiangolo/full-stack-fastapi-template/pull/1192
